setup.sh

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
What is the purpose of this script? A description to the beginning could be useful.